pelicanconf.py

```diff
@@ -31,6 +32,12 @@
 
 import fnmatch
 import os
+import json
+
+jsonFile = open(DOWNLOADS_JSON_PATH)
+DOWNLOADS = json.load(jsonFile)
+jsonFile.close()
```

Comment:
What if the file do not exists? Could we add a try/catch block here?

Reply:
If the file doesn't exist it will throw a Python error in the console and stop the site from building:
```
CRITICAL: IOError: [Errno 2] No such file or directory: u'downloads.json'
Makefile:28: recipe for target 'html' failed
make: *** [html] Error 1
```
We could have a try catch with our own messages for different kinds of exceptions, but is there an advantage?